docs_source_files/content/getting_started/quick.en.md

```diff
@@ -8,7 +8,7 @@ but it composes many tools.
 
 ## WebDriver
 
-If you are beginning with desktop website test automation, then you
+If you are beginning with desktop website or mobile website test automation, then you
```

Comment:
```suggestion
If you are beginning with website test automation, then you
```

Reply:
@diemol  
I run Selenium Test cases for both the mobile site and desktop website inside the chrome browser, so I wanted this text to be more generic.

Reply:
Exactly, that's why we can remove `desktop` and just say `websites`, which should cover everything, don't you think?

Reply:
@diemol  
okay, but I have seen lots of Automation testers that they don't know the fact that they too can automate the mobile websites inside the chrome browser when their app uses the common web views.

This could be helpful to the entire community.

Reply:
Fair enough, let's leave it as you proposed, thanks!

Reply:
@diemol  Thanks